ext/date/php_date.stub.php

```diff
@@ -662,6 +662,18 @@ public static function __set_state(array $array): DateTimeZone {}
 
 class DateInterval
 {
+    public ?int $y;
```

Comment:
The manual states that ` available properties listed below depend on PHP version, and should be considered as readonly.`, so I'm wondering whether we want to make these actually `readonly`. 🤔